src/ipc-handlers.ts

```diff
@@ -909,12 +909,24 @@ export async function showErrorMessageBox(
 		/Error invoking remote method '\w+': Error:/g,
 		''
 	);
-	await showMessageBox( event, {
+
+	console.error( filteredError );
+
+	const response = await showMessageBox( event, {
 		type: 'error',
 		message: title,
-		detail: error ? `${ message }\n\nError: ${ filteredError }` : message,
-		buttons: [ __( 'OK' ) ],
+		detail: message,
+		buttons: [ __( 'Open Studio logs' ), __( 'OK' ) ],
```

Comment:
Should we display the "Open Studio logs" buttons in all the cases? or just when we receive the "errorToLog" parameter?

Reply:
Good point! We could use a param, e.g. `showOpenLogs`, that is set to `false` by default, and we can enable it per the case.